tests/core/test_channels.py

```diff
@@ -869,7 +869,7 @@ async def test_slackbot_send_text():
 
 @pytest.mark.filterwarnings("ignore:unclosed.*:ResourceWarning")
 def test_channel_inheritance():
-    from rasa.core.channels.channel import RestInput
+    from rasa.core.channels import RestInput
     from rasa.core.channels.rasa_chat import RasaChatInput
```

Comment:
why do we the import from `channels` instead of `channels.rest`? Maybe this should be consistent among the way we import channels in the tests

Reply:
the way I see this public API is that if you want to use a channel, you should import from the `rasa.core.channels` modules. You do not really need to know how the `channels` module is organised when you're consuming it.